includes/class-image-optimisation.php

```
@@ -32,15 +32,20 @@ class Image_Optimisation {
+	 */
+	private array $options;
 
-		/**
-		 * Configuration options for image optimization.
-		 *
-		 * @var array
-		 * @since 1.0.0
-		 */
-		private array $options;
+	private array $exclude_convert_imgs = array();
+	private array $preload_front_page_urls = array();
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $preload_front_page_urls. Keep variable name length under 20. (undefined)

(LongVariable)